docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: "3.3"
+services:
```

Comment:
I saw you are using a python method to get and display the bb report, but you don't set up the service here?

Reply:
I thought setting up a service was made once you name it as I did below " db_service." Is that not right?